Makefile

```diff
@@ -289,26 +317,58 @@ git-diff-check:
 
 RPM_SPECFILE=rpmbuild/SPECS/osbuild-$(COMMIT).spec
 RPM_TARBALL=rpmbuild/SOURCES/osbuild-$(COMMIT).tar.gz
+RPM_SPECFILE_DEVEL=rpmbuild/SPECS/osbuild-SRC.spec
+RPM_TARBALL_DEVEL=rpmbuild/SOURCES/osbuild-SRC.tar.gz
 
 $(RPM_SPECFILE):
 	mkdir -p $(CURDIR)/rpmbuild/SPECS
 	(echo "%global commit $(COMMIT)"; git show HEAD:osbuild.spec) > $(RPM_SPECFILE)
 
+$(RPM_SPECFILE_DEVEL): osbuild.spec
```

Comment:
Could you please add a comment why this rpm is build differently from the other rpm, i.e. why do we need two?

Reply:
because it states that it's built from source trying to avoid any semblance from a "release rpm"
I can add this as a comment.